Sunrise/Helpers/AppRouting.swift

```diff
@@ -3,18 +3,38 @@
 //
 
 import UIKit
+private func < <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
+  switch (lhs, rhs) {
+  case let (l?, r?):
+    return l < r
+  case (nil, _?):
+    return true
+  default:
+    return false
+  }
+}
```

Comment:
Cleaned it up. Here's a proposal to remove this, in case you're interested:
https://github.com/apple/swift-evolution/blob/master/proposals/0121-remove-optional-comparison-operators.md